integration_tests/tests/consistency/consistency_user_unsubscribes.sql

```diff
@@ -0,0 +1,52 @@
+
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
```

Comment:
Since these tests were written in order to test the old version of the connector, would you be please leave a comment in these files to mention for future use that these tests expect the `iterable_user_unsubscribed_channel_identifier` and  `iterable_user_unsubscribed_message_type_identifier` variables to be commented out in order for the validations to run properly. 

Further, could you renamed these validations to clearly label them as validations for the pre Aug 2023 connector updates. We will then want to create additional validations for the post Aug 2023 connector updates in the future.

Reply:
Added.